docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -1499,6 +1501,19 @@ If a task in a multi-host task group is restarted:
   a new task group, so it will run the teardown group commands, clear the task
   directory, and re-run the setup group commands.
 
+#### TearDown task and teardown group reliability 
```

Comment:
nit -- can we remove the camelcase for TearDown task? This seems inconsistent.

Reply:
This also seems to have resolved itself. 
I just tried running the problematic task again so that I can monitor the host in the aws dashboard as its running, but the host remained okay and the teardown group ran. 🤷🏻

Reply:
Do you have If we don't run teardown group until another task runs, maybe it's not that there's an error with the host. For example, this host https://spruce.mongodb.com/host/i-0afafb32da357d10a may have just been decommissioned because it was idle (Bynn increased the idle time here, so it's possible this is less likely to happen now). Does that job check to make sure that we don't decommission without running teardown group?

Reply:
yes, that ended up being the reason. I found a honeycomb trace that showed a teardown group starting to run teardown but not finishing. I was also able to recreate it in staging. 

I removed the part of the doc saying that teardown_group is more likely to run into issues than teardown_task. With the fix I don't know how much of a difference it would be.